lib/common/acl.c

```diff
@@ -118,14 +122,25 @@ __xml_acl_create(xmlNode *xml, GList *acls, enum xml_private_flags mode)
 
             CRM_LOG_ASSERT(offset > 0);
             acl->xpath = strdup(buffer);
-            crm_trace("Built xpath: %s", acl->xpath);
+            crm_trace("Unpacked ACL <%s> element using equivalent xpath: %s",
+                      crm_element_name(xml), acl->xpath);
```

Comment:
5. minor, "equivalent" was rather surprising for me, would expect e.g. "synthetized"